src/Components/CarPage/CarDetail.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Grid, Typography, Rating, Link, Button } from '@mui/material'
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import IconButton from '@mui/material/IconButton';
+
+
+const CarDetail = ({ name = "car", type = "Sport", capacity = 2, steering = "Manual", gasoline = 70, price = 100, discountedPrice, info = "NISMO has become the embodiment of Nissan's outstanding performance, inspired by the most unforgiving proving ground, the 'race track'.", reviews = 100, rating = 5, liked = true }) => {
```

Comment:
what your plan to handle this data after working with an api ?

Reply:
It will recieve an object which has all these props, instead of passing all of them